src/test/resources/projects/skip/bom.xml

```diff
@@ -0,0 +1,3204 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Straight copy from the projects/run/bom.xml.  no differences.

Reply:
This file isn't needed.  I'd rather it was included in the merge as it will just need to be removed.